controllers/addresspool_controller.go

```diff
@@ -43,9 +41,13 @@ type AddressPoolReconciler struct {
 	Namespace string
 }
 
-const RetryPeriod = 5 * time.Minute
+const (
+	RetryPeriod                       = 5 * time.Minute
+	MetalLBConfigMapName              = "config"
+	AddressPoolManifestPathController = "./bindata/configuration/address-pool"
+)
 
-var AddressPoolManifestPath = "./bindata/configuration/address-pool"
+var AddressPoolManifestPath = AddressPoolManifestPathController
```

Comment:
why you can't just remove this var?

Reply:
we can't remove it this var is used by renderObject, this var is initialized differently if u running production operator it will be set to `AddressPoolManifestPathController` and if u running controller unit-test it will be set to `AddressPoolManifestPathControllerTest` then it get reset back to the original value after the test is done